data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -57,45 +69,52 @@ def extract(self) -> None:
         logger.info("Reading US GeoJSON (~6 minutes)")
         self.geojson_usa_df = gpd.read_file(
             self.CENSUS_USA_GEOJSON,
-            dtype={"GEOID10": "string"},
```

Comment:
I always get nervous around free-floating strings, so I went ahead and replaced these with constants.